src/qml/AboutPage.qml

```diff
@@ -16,37 +17,78 @@
  */
 
 import QtQuick 2.9
+import org.asteroid.utils 1.0
 import org.asteroid.controls 1.0
 import org.nemomobile.systemsettings 1.0
 
-Item {
+Flickable {
     AboutSettings {
         id: about
     }
-
-    Icon {
-        name: "logo-asteroidos"
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.topMargin: Dims.h(10)
-        anchors.top: parent.top
-        anchors.bottom: osLabel.top
-        anchors.bottomMargin: Dims.h(3)
-        width: height
-    }
-    Label {
-        id: osLabel
-        text: about.operatingSystemName
-        font.bold: true
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.bottom: releaseLabel.top
+    DiskUsage {
+        id: diskUsage
     }
-    Label {
-        id: releaseLabel
-        text: about.softwareVersion
-        opacity: 0.8
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.bottom: parent.bottom
-        anchors.bottomMargin: Dims.h(7)
+
+    contentHeight: contentcolumn.implicitHeight
+    Column {
+        id: contentcolumn
+        anchors.fill: parent
+        Item { //this acts as a spacer to put the logo in the middle of the screen.
+            height: parent.width*0.1
+            width: height
+        }
+        Icon {
+            name: "logo-asteroidos"
+            anchors.horizontalCenter: parent.horizontalCenter
+            width: parent.width*0.4
+            height: width
+        }
+        Label {
+            id: osLabel
+            text: about.operatingSystemName
+            anchors.horizontalCenter: parent.horizontalCenter
+        }
+        Label {
+            id: releaseLabel
+            text: about.softwareVersion
+            opacity: 0.8
+            anchors.horizontalCenter: parent.horizontalCenter
+        }
+        Repeater {
+            model: [
```

Comment:
Great :)

Reply:
I was pleased with how this turned out, thanks!